ci/scripts/jenkins/build.sh

```diff
@@ -105,16 +99,14 @@ gpuci_logger "sccache usage for morpheus build:"
 sccache --show-stats
 
 gpuci_logger "Installing Morpheus"
-pip install -e ${MORPHEUS_ROOT}
+cmake -DCOMPONENT=Wheel -P ${MORPHEUS_ROOT}/build/cmake_install.cmake
+pip install ${MORPHEUS_ROOT}/build/wheel
 
 gpuci_logger "Archiving results"
-mamba pack --quiet --force --ignore-editable-packages --ignore-missing-files --n-threads ${PARALLEL_LEVEL} -n morpheus -o ${WORKSPACE_TMP}/conda_env.tar.gz
-tar cfj ${WORKSPACE_TMP}/workspace.tar.bz --exclude=".git" --exclude="models" --exclude=".cache" ./
-ls -lh ${WORKSPACE_TMP}/
+mamba pack --quiet --force --ignore-missing-files --n-threads ${PARALLEL_LEVEL} -n morpheus -o ${WORKSPACE_TMP}/conda_env.tar.gz
```

Comment:
Because without it the test & documentation stages won't have the dependencies installed. Its faster to just archive the entire conda env than it is for the other stages to have to rebuild it.

Reply:
> Its faster to just archive the entire conda env than it is for the other stages to have to rebuild it.

This is surprising. Even though it takes longer, there is some value to installing the morpheus package into a freshly created environment. Do you have an idea on how much of a time difference there is between extracting the environment and recreating it?

Reply:
I'm assuming that building the conda env in the test & documentation stages would take roughly the same time as the build stage. 

Looking at the most recent Jenkins run it took 9m41s to build the conda env and 2m18s to archive and transfer it to S3.
It took the test stage 3m6s to download and extract the env.

A savings of about 6 minutes. Some of that archive/transfer time would diminish if we instead archived the wheel dir which would be much smaller than the entire conda env.

My concern would be the risk that the build env and the test env wouldn't be identical, and there is a potential of a new conda package being published between the build and test stages.

Reply:
>My concern would be the risk that the build env and the test env wouldn't be identical, and there is a potential of a new conda package being published between the build and test stages.

This will actually replicate more what normal users will experience. So I am not explicitly against allowing this to happen.

Reply:
Going to resolve this for now. But we should keep an eye on it.